### PR TITLE
display-ghi-information

### PR DESCRIPTION
**Pull request by sirtobiwan**
when you click on the ghi information, a new window with more information should pop up

### CODE REVIEW

File: components/Card/index.js [hunk 1]
```diff
@@ -42,6 +90,27 @@ export default function Card() {
...
+          <Modal
+            text={
+              <>
+                <FontAwesomeIcon icon={faAngleDown} /> radiation in ghi
+               </>
+                }
+            modalContent={
+              <>
+                <h2>Radiation in GHI </h2>
+                <p>
```
Comment (rabe200) [inline components/Card/index.js:103]: use a section here instead of paragraph for better accessibility

File: components/Card/index.js [hunk 2]
```diff
@@ -42,6 +90,27 @@ export default function Card() {
...
+        <StyledRadiation>
+          <Modal
+            text={
+              <>
+                <FontAwesomeIcon icon={faAngleDown} /> radiation in ghi
+               </>
+                }
+            modalContent={
+              <>
+                <h2>Radiation in GHI </h2>
```
Comment (rabe200) [inline components/Card/index.js:102]: if you use h2 , you should have a h1 before. don't use headlines for their font-size only. instead create a styled - component and change the font-size there

File: components/Card/index.js [hunk 3]
```diff
@@ -42,6 +90,27 @@ export default function Card() {
...
+                <p>
+                  Global Horizontal Irradiance (GHI) is the total solar
+                  radiation incident on a horizontal surface. It is the sum of
+                  Direct Normal Irradiance (DNI), Diffuse Horizontal Irradiance,
+                  and ground-reflected radiation.
+                </p>
+                <StyledImage src="/ghi.gif" alt="description" />
```
Comment (rabe200) [inline components/Card/index.js:109]: use next/image : styled(Image)
then add width and height props 
this helps to increase the performance of image rendering

**Review by rabe200 — CHANGES_REQUESTED** [review]
looking good so far - but see the comments :)

File: components/Card/index.js [hunk 4]
```diff
@@ -1,4 +1,8 @@
 import styled from "styled-components";
+import Modal from "@/components/Modal";
```
Comment (Satttoshi) [inline components/Card/index.js:2]: using @ as a root prefix, might be an issue if you start addressing libraries, which has the same prefix.

For example @next/fonts

Using "~" is more common for referring roots in a nextjs project

File: components/Card/index.js [hunk 5]
```diff
@@ -30,6 +35,49 @@ const StyledData = styled.p`
...
+  border-radius: 5px;
+  font-size: 1.2rem;
+  padding: 0.75rem 1.5rem;
+  cursor: pointer;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.2);
+  transition: background-color 0.3s ease;
+  text-decoration: none;
+
+  &:hover {
+    background-color: #e6951d;
```
Comment (Satttoshi) [inline components/Card/index.js:73]: Hover effect only works on Dekstop properly, make sure to aim for "Mobile first".
To get the desired effect you want, you can try this:

```css
  @media (hover: hover) {
    &:hover:enabled {
      background-color: #e6951d);
    }
  }

  @media (hover: none) {
    &:active:enabled {
      background-color: #e6951d;
    }
  }
 ```

Comment (Satttoshi) [reply components/Card/index.js:73]: second media query applies to "tapping" on mobile

File: components/Modal/index.js [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+        </Overlay>
+      )}
+    </>
+  );
+}
+
+const Overlay = styled.div`
+  color: black;
+  position: fixed;
+  z-index: 999;
```
Comment (Satttoshi) [inline components/Modal/index.js:33]: Overlay is only sibling to your div, 999 might be overkill on 2 elements. Remember z-index only applies to sibling elements!

**Review by Satttoshi — APPROVED** [review]
Good job, nice simple Modal.

LGTM :rocket: 

anyways please check some of my comments.

**Pull request merged**